### DISCUSSION TITLE
Config Management Plugin not called for helm charts

### DISCUSSION THREAD

**Discussion by OlivierJavaux** [argoproj/argo-cd#11824] [Q&A] [answered]
Hello,

I am using argocd 2.5.5 (latest release). Following the doc, I have added a Config Management Plugin using sidecar.
It is working as expected for yaml files.
I adapted it for helm charts but it is not called for helm charts. Discovery is not called. Nothing in the plugin container logs. I have added debug traces in the discover command and I can confirm that it is not called.
Did I miss something?

Thanks for your help,

                  Olivier

**Comment by OlivierJavaux** [comment, marked as answer]
I have the answer: in my application yaml file, I have:
```
  source:
    helm:
      valueFiles:
      - values-demo.yaml
```
My understanding is that ArgoCD sees it is helm chart, and uses the builtin helm plugin, avoiding my own plugin to be used.
If I remove this part, my plugin is correctly used.

**Reply by saplaum** [reply]
Ran into the same situation and found - after several frustrating hours - this post. Thank you for sharing this, Olivier!
